api/src/wfl/service/datarepo.clj

```diff
@@ -228,8 +228,7 @@
   (let [bq-name (bigquery-name dataset)
         query   "SELECT %s
                  FROM   `%s.%s.%s`
-                 WHERE  datarepo_ingest_date > '%s'
-                 AND    datarepo_ingest_date <= '%s'"]
+                 WHERE updated BETWEEN '%s' AND '%s'"]
```

Comment:
I'd like to not hard-code this field as no doubt it'll be different in the real dataset. Can you pass in the name of the field instead?

Reply:
`wfl.util/today` and so on still return a `LocalDate`?

Reply:
Yes i think so. I thought they returned a `LocalDate` at UTC+0

Reply:
How does that fit with the change in datarepo_test.clj?

Reply:
Guess it doesn't?

Reply:
Seems WFL should either use datetime or timestamp,
or query-table-between should take date,
but it works if it works.
(And test-create-snapshot passes.)

Reply:
I think the idea is that you give it an interval of the same time as the field.

Reply:
I resolved the original ask, but am confused on the conversation direction. Does this mean that we should be able to take in both inputs (date and datetime)?

Reply:
Presumably it should take whatever datatype BQ SQL's `BETWEEN` syntax expects.
I saw the interval to query-table-between constructed with `util/today` and so on.
`util/today` returns a `LocalDate` but it looks like query-table-between takes `DATETIME`s,
(or something) at least according to the `datarepo_test.clj` change below.
(But maybe I'm confused again about how to handle time in queries.)

Reply:
I think `wfl.tools.snapshots/create-snapshots` is an outdated test now as we don't want to use DATEs anymore. (that test is not used anyways). Maybe that's where @tbl3rd 's confusion came from?